engines/azure/app/models/azure/resource_group.rb

```diff
@@ -0,0 +1,31 @@
+module Azure
+  class ResourceGroup < AzureResource
+    attr_accessor :name
+
+    def azure_create()
+      self.creation_attributes = {
+        name: @name
+      }
+      response = JSON.parse(@cli.create_resource_group(name: @name))
+      self.id = response['id']
+      self.refresh()
+    end
+
+    def ready!
+      self.wait_until('Succeeded')
+    end
+
+    def azure_destroy()
+      @cli.destroy_resource_group(name: @name)
```

Comment:
do we not want to generate script to download for the user with the command ?

Reply:
A one liner is IMHO not worth doing anything extra.